mlir/include/mlir/Dialect/MIOpen/MIOpen.h

```diff
@@ -259,6 +267,10 @@ class BottomUpCTBuilder : public CoordTransformsBuilder {
   // Defines a dimension that is not mapped to any coordinates in the output
   void addDim(StringRef name, uint32_t dim, int64_t size);
 
+  void expand(ArrayRef<uint32_t> dims, ArrayRef<int64_t> sizes);
```

Comment:
expand() still feels too specific

Reply:
What I'd suggest is the following: `passThrough(ArrayRef<uint32_t> endIndices, ArrayRef<uint32_t> startIndices)` - this lets you avoid having to talk about the names at all